pkg/fission-cli/cmd/spec/validate.go

```diff
@@ -72,6 +78,102 @@ func (opts *ValidateSubCommand) run(input cli.Input) error {
 	return nil
 }
 
+// ResourceExists checks if the spec resources exists in the cluster
+func ResourceExists(opts *ValidateSubCommand, fr *FissionResources) error {
+	result := utils.MultiErrorWithFormat()
+
+	fnlist, err := GetAllFunctions(opts.Client())
+	if err != nil {
+		return errors.Errorf("Unable to get Functions %v", err.Error())
+	}
+	for _, f := range fnlist {
+		for _, fn := range fr.Functions {
+			// if the function name is same check if they are present in same namespace
+			if f.Metadata.Name == fn.Metadata.Name && f.Metadata.Namespace == fn.Metadata.Namespace {
+				result = multierror.Append(result, fmt.Errorf("\nFunction %v/%v already exist", fn.Metadata.Name, fn.Metadata.Namespace))
```

Comment:
Remove the `\n` in all error messages. otherwise, the message will look like following.
```
Options
  --specdir=''   Directory to store specs, defaults to ./specs
  --server=''     Server URL
  --verbosity=1   CLI verbosity (0 is quiet, 1 is the default, 2 is verbose)

Usage:
  fission spec validate [flags] [options]

Use "fission spec options" for a list of global command-line options (applies to all commands).

Error:  Spec validation failed: 3 errors occurred:
        * 
Function hello/default already exist
        * 
Environment python-27/default already exist
        * 
Pakcage hello-pkg/default already exist
```

(here and other places)

Reply:
`already exist` -> `already exists`  (here and other places)